coding-concepts.js

```diff
@@ -8,36 +8,36 @@
 // --------------------1) What will this log?
 
 const cohort = "Hotel 2022"
-// console.log(cohort.split(""))
+console.log(cohort.split(""))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: [hotel , 2022] because (.split()) method turns strings into arrays
+// b) Verify and explain: ['H', 'o', 't', 'e','l', ' ', '2', '0','2', '2' ] i was kinda right turns the string into the array but split every single letter
 
 // --------------------2) What will this log?
 
 const greeter = (name) => {
   `Hello, ${name}!`
 }
-// console.log(greeter("LEARN Student"))
+console.log(greeter("LEARN Student"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer:`Hello,"LEARN Student"!` the greater is reuseable and you can maniuplate it into say any with the string interpolation
+// b) Verify and explain: undefined - im at a loss, im guessing greeter is in the wrong input
 
 // --------------------3) What will this log?
 
 const multipliedByTwo = [4, 5, 6, 7, 8].map((number) => number * 2)
-// console.log(multipliedByTwo)
+console.log(multipliedByTwo)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: [8, 10, 12, 14, 16] the numbers are .map so the values in the array can be changed
+// b) Verify and explain: [8, 10, 12, 14, 16] as i stated .map can change the values in the array
```

Comment:
yes .map allow logic to occur on each value in the array. .map will return a another array as the same length as the original array